README.md

```diff
@@ -111,11 +194,11 @@ $ skopeo delete docker://localhost:5000/imagename:latest
 
 Private registries with authentication
 -
-When interacting with private registries, `skopeo` first looks for `--creds` (for `skopeo inspect|delete`) or `--src-creds|--dest-creds` (for `skopeo copy`) flags. If those aren't provided, it looks for the Docker's cli config file (usually located at `$HOME/.docker/config.json`) to get the credentials needed to authenticate. The ultimate fallback, as Docker does, is to provide an empty authentication when interacting with those registries.
+When interacting with private registries, `skopeo` first looks for `--creds` (for `skopeo inspect|delete`) or `--src-creds|--dest-creds` (for `skopeo copy`) flags. If those aren't provided, it looks for the container authorization config file (usually located at `$XDG_RUNTIME_DIR/containers/auth.json`) to get the credentials needed to authenticate. The ultimate fallback, as other container engines do, is to provide an empty authentication when interacting with those registries.
```

Comment:
Being completely silent about `~/.docker/config.json` is not helpful.

Reply:
I would think this is more a man page information rather then a README.md

Reply:
To a large extent, most of this README duplicates the man page… especially this kind of detail should be only in there.

The way I think about it, the README can omit features and properties, but if we decide to say something, it should be _correct_.

Ultimately, the users don’t care and shouldn’t care about the file paths. What do you think about using just
> `skopeo` uses credentials from the `--creds` (for `skopeo inspect|delete`) or `--src-creds|--dest-creds` (for `skopeo copy`) flags, if set; otherwise it uses configuration set by `skopeo login`, `podman login`, `buildah login`, or `docker login`.

and dropping both the example and the final paragraph? We don’t want users to spend time `cat`ing `auth.json`, so this no longer serves much purpose in the top-level README.

Sure, it still refers to Docker — but it is so crowded in the list now :)

Reply:
SGTM